formats/financial_release/publisher/details.json

```diff
@@ -0,0 +1,26 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "additionalProperties": false,
+   "required": [
+     "title",
+     "sub_title",
+     "content",
+     "published_date"
+   ],
+  "properties": {
+    "title": {
+      "type": "string"
+    },
+    "sub_title": {
```

Comment:
In other areas of GOV.UK we refer to this as a `summary`, worth keeping to the same naming scheme where possible. This is what would render a press release in Whitehall: https://github.com/alphagov/whitehall/blob/master/app/views/news_articles/show.html.erb
And the model for it if it's useful: https://github.com/alphagov/whitehall/blob/master/app/models/news_article.rb.

My bad, I should have linked you these before.